lua/treesitter-context/render.lua

```diff
@@ -150,8 +150,12 @@ local function highlight_contexts(bufnr, ctx_bufnr, contexts)
         if nsrow >= start_row then
           local msrow = offset + (nsrow - start_row)
           local merow = offset + (nerow - start_row)
-
-          local hl = buf_query.hl_cache[capture]
+          local hl --- @type integer
+          if buf_query.get_hl_from_capture then
```

Comment:
Is this not always defined?

Reply:
It would be undefined on older neovim versions.

Reply:
Which versions? We only target the latest release.

Reply:
I think not breaking older versions is nice, it can be refactored later. In my other plugins I usually try to have them working on whatever debian/ubuntu stable has, which is 0.7 at the moment.

Reply:
Not for a treesitter plugin. Even for Neovim core, treesitter is still considered experimental and the API's are moving quickly.

We can keep this as is, but just note going forward, backwards compatibility is a non-goal.

Reply:
Hi. I have lost access to my account @Sam-programs.

 > Which versions? We only target the latest release.

The function isn't defined in stable, tag latest in the releases page.